src/Symfony/Component/HttpKernel/DataCollector/LoggerDataCollector.php

```diff
@@ -171,6 +198,9 @@ private function computeErrorsCount()
             if ($this->isSilencedOrDeprecationErrorLog($log)) {
                 if ($log['context']['exception'] instanceof SilencedErrorContext) {
                     ++$count['scream_count'];
+                } elseif (isset($log['context'][BufferingLogger::BOOTSTRAPPING_FLAG])) {
```

Comment:
There is already a dependency to the debug component on the `SilencedErrorContext`. I understand that it create a (second) hard dependency between those components, a hard coded key would be better?